andela/ah-the-unsullied-frontend
# #162949179 List articles and paginate

#### What does this PR do?

-List all articles
- Paginate articles feed (ten articles per page)


#### Description of Task to be completed?
- [x] User views ten articles per page 

#### How should this be manually tested?

1. Clone the repository
```
git clone https://github.com/andela/ah-the-unsullied-frontend.git
```
2. Cd into the cloned repo
```
cd ah-the-unsullied-frontend
```
3. Checkout to this branch
```
git checkout ft-paginate-articles-162949179
```
4. Install the dependencies of the project
```
npm install
```
5. Run the app
```
npm start
```
6. Visit the default route `http://localhost:3000/` to see all the articles
- Press go to the bottom of the page and test out pagination

7. To run tests, run `npm test`

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#162949179](https://www.pivotaltracker.com/story/show/162949179)

#### Screenshots (if appropriate)
<img width="1196" alt="screenshot 2019-02-21 at 02 06 56" src="https://user-images.githubusercontent.com/33450849/53131328-8d573e80-357d-11e9-9192-44bbd47126d5.png">

#### Questions:
